Makefile

```diff
@@ -213,6 +213,7 @@ format:
 .PHONY: check
 check:
 	@GOARCH=amd64 GOOS=dragonfly go build ./... # Check if the internal/platform can be built on compiler-unsupported platforms
+	@GOARCH=386 GOOS=linux go build ./... # Check if the internal/platform can be built on compiler-unsupported platforms
```

Comment:
this is the missing check, though theoretically we could fail on the various flavors of 32-bit arm or powerpc